public/scripts/home.js

```diff
@@ -0,0 +1,122 @@
+$(function () {
+    window.App = {
+        Models : {},
+        Views : {}
+    };
+
+    window.template = function (id) {
+        return _.template( $(id).html() );
+    };
+
+    App.Models.StyleModel = Backbone.Model.extend ({
+        defaults : {
+            'fontSize' : '14px',
+            'color' : '#000000',
+            'background' : '#ffffff'
+        },
+        urlRoot : '/getSettings'
+    });
+
+    App.Views.StyleView = Backbone.View.extend ({
+        tagName : 'style',
+
+        template : template(styleTemplate),
+
+        render : function (){
+            var template = this.template(this.model.toJSON());
+            this.$el.html( template );
```

Comment:
this can be done shortly like: `this.$el.html( this.template(this.model.toJSON()) );`